web/src/main/java/org/cbioportal/proxy/ProxyController.java

```diff
@@ -18,24 +18,32 @@
 
 @RestController
 public class ProxyController {
+    private static final String DEFAULT_ONCOKB_URL = "https://www.oncokb.org/api/v1";
+    
     @Value("${show.oncokb:true}")
     private Boolean showOncokb;
     @Value("${oncokb.token:}")
     private String oncokbToken;
-    @Value("${oncokb.public_api.url:www.oncokb.org}")
-    private String oncokbPublicApiUrl;
+    @Value("${oncokb.public_api.url:}")
+    private String oncokbApiUrl;
 
     private Logger LOG = LoggerFactory.getLogger(ProxyController.class);
 
     @RequestMapping("/**")
     public String proxy(@RequestBody(required = false) String body, HttpMethod method, HttpServletRequest request)
         throws URISyntaxException {
         HttpHeaders httpHeaders = initHeaders(request);
-        return exchangeData(body, buildUri(request.getPathInfo(), request.getQueryString(), false), method, httpHeaders).getBody();
+        
+        return exchangeData(body,
+            buildUri(request.getPathInfo(), request.getQueryString(), false),
+            method,
+            httpHeaders,
+            String.class
+        ).getBody();
     }
 
     @RequestMapping("/oncokb/**")
-    public ResponseEntity<String> proxyOncokb(@RequestBody(required = false) String body, HttpMethod method, HttpServletRequest request)
+    public ResponseEntity<Object> proxyOncokb(@RequestBody(required = false) String body, HttpMethod method, HttpServletRequest request)
```

Comment:
@onursumer what was the error by using `String` return type?

Reply:
@kalletlak content is attached to `response.text` instead of `response.body` if we use `String`, and that causes OncoKB frontend API client to always return an empty response.